.travis.yml

```diff
@@ -38,3 +39,6 @@ script:
     - bin/console doctrine:migrations:migrate --no-interaction
     - vendor/bin/behat --profile=smoke -fprogress
     - popd
+
+    # Shell script tests
+    - for file in $(find . -type f \( -perm +111 -or -name "*.sh" \) ! -path "*/vendor/*" ! -path "*/.git/*" ! -path "*/api/bin/console" ! -path "*/api/bin/symfony_requirements" | sed 's#^\.#/app#'); do docker run -v "$(pwd):/app" koalaman/shellcheck:v0.4.7 --exclude SC1091 "$file"; done
```

Comment:
As this is fast, could you move it after the PHPUnit tests?

Reply:
Sorted - also tried to ensure docker is available as the build failed